### ISSUE TITLE
Lose range

### ISSUE THREAD

**Issue by patchido** [maniacbug/RF24Network#5]
What happens if i lose the range in the network? if it is regained will it start communicating again?

**Comment by shulter** [comment]
Yes. However, packets which experienced a transmit timeout and could not be transmitted even after the predefined retry count are discarded.
